boards/Teensypp20/bin/Teensypp20.ino.hex

```diff
@@ -1,320 +1,320 @@
-:10000000B3C00000D9C00000D7C00000D5C00000B8

```

Comment:
I think, there's a problem here... Didn't you want to add ArduinoNano.ino.hex here instead of a Teensypp20.ino.hex when Teensypp20.ino hasn't been even touched?

Reply:
I moved boards/bin/Teensypp20.ino.hex into the /boards/Teensypp20/bin directory, but now I realize that it fine at the location where it was. So I'll revert that file move, and also add ArduinoNano.ino.hex into the boards/bin folder

Reply:
I moved Teensypp20.ino.hex back to the original location, and added the .hex files for the ArduinoNano (one with and one without bootloader, as some Nano versions have their own bootloader version)